src/components/pages/todo/TodoList.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { ChangeEvent, useState, useEffect } from 'react';
+import Checkbox from '@mui/material/Checkbox';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import Grid from '@mui/material/Grid';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import { DatePicker } from '@mui/x-date-pickers';
+import { format } from 'date-fns';
+import { Task } from '../../../interface/interface';
+import { styled } from '@mui/system';
+
+const defaultPlantList =
+  'https://hortology.co.uk/cdn/shop/files/Ficus-elastica-Melany-Rubber-Plant-14x45cm-Hadleigh-Plant-Pot-White-20x17.5cm_52335388-022e-4750-9c9f-54efbba9ea0a_1200x.jpg?v=1704197517';
+
+const checkImageUrl = (imageUrl: string): boolean => {
+  try {
+    new URL(imageUrl);
+    return true;
+  } catch (err) {
+    return false;
+  }
+};
+
+const ListElement = styled('div')({
+  display: 'flex',
+  alignItems: 'center',
+  marginBottom: '16px',
+  padding: '12px',
```

Comment:
better to use rems

Reply:
got used to pixels after my last project, I'll keep that in mind